container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/logs_aggregator_functions/implementations/vector/vector_configuration_creator.go

```diff
@@ -0,0 +1,228 @@
+package vector
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/go-yaml/yaml"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/shared_helpers"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_impls/kubernetes/kubernetes_manager"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_impls/kubernetes/object_attributes_provider"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/uuid_generator"
+	"github.com/kurtosis-tech/stacktrace"
+	"github.com/sirupsen/logrus"
+	apiv1 "k8s.io/api/core/v1"
+)
+
+const (
+	configuratorContainerName = "logs-aggregator-configurator"
```

Comment:
it seems like this container is actually validating the config, not creating it? (creation happens by creating the config map) - what do you think about renaming `configurator` to `validator`?

Reply:
Yes can do